specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/preview/2020-07-01-preview/azureADMetrics.json

```diff
@@ -0,0 +1,443 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2020-07-01-preview",
+    "title": "azureactivedirectory",
+    "description": "Metrics and Alerts for Azure Active Directory."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/microsoft.aadiam/azureADMetrics/{azureADMetricsName}": {
+      "put": {
+        "tags": [
+          "azureADMetrics"
+        ],
+        "operationId": "azureADMetrics_Create",
```

Comment:
>Create [](start = 39, length = 6)

CreateOrUpdate

Reply:
does this mean to have the operationId as CreateOrUpdate?

Reply:
Currently, the create and update operations are separate

Reply:
Yes, a PUT is both a Create and Update operation.  The convention is to use _CreateOrUpdate for PUT operations.  Your PATCH operation being just _Update is correct.